apps/sms/views/conversation/js/conversation.js

```diff
@@ -2676,7 +2675,7 @@ var ConversationView = {
       return;
     }
 
-    if (!contacts || !contacts.length) {
```

Comment:
Looks like ```ConversationView.searchContact``` can still return ```null```, you need to take care about this method too - make it to return empty array as well.